features/hbx_admin/step_definitions/ui_config_steps.rb

```diff
@@ -32,4 +32,117 @@
   expect(page).to_not have_content("Inbox")
 end
 
+Given(/notices feature is enabled?/) do
```

Comment:
https://github.com/ideacrew/enroll/blob/6570ae150b254e35cda2a5d40bbd3cb77a48abf1/features/support/worlds/resource_registry_world.rb#L40 We can get rid of these steps, lets use it dynamically. You'll just have to write in your cucumber something like:

`And EnrollRegistry notices_tab feature is enabled`